README.md

```diff
@@ -134,7 +134,10 @@ Read more about [Netlify redirects](https://docs.netlify.com/routing/redirects/)
 
 ### Versions
 
-You can check our `package.json` for supported Next and Node versions. Our support of Next 10 is currently experimental.
+You can check our `package.json` for supported Next and Node versions.
```

Comment:
Where do our `package.json` says the supported Next version? Is it here https://github.com/netlify/netlify-plugin-nextjs/blob/ff311a06991f3fc98b4f026c138137ecdfff9ad8/package.json#L68

Maybe it's better to explicitly write the the min supported version?